### PR TITLE
Claidheamh Mòr

### PR DESCRIPTION
**Pull request by StarInAPond**
Let's demoman with boots to survive 1 hit carrying this weapon, there's no reason to ever use it over other melees otherwise

### CODE REVIEW

**Review by FortyTwoFortyTwo — COMMENTED** [review]
Just 5% sounds like too little of a effect that it'll turn this weapon into a direct upside, maybe replace penalty to something else?

File: addons/sourcemod/configs/vsh/vsh.cfg [hunk 1]
```diff
@@ -1334,7 +1334,9 @@
 		
 		"327"	//Claidheamh Mòr
 		{
-			"desp"			"Claidheamh Mòr: {positive}Refills 25% of your charge meter on hit"
+			"desp"			"Claidheamh Mòr: {positive}Refills 25% of your charge meter on hit, {neutral}replaces damage vulnerability with slower attack speed"
+			"attrib"		"5 ; 1.15"
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/configs/vsh/vsh.cfg:1338]: `412 ; 1.0` need to be readded so replaces damage vulnerability actually comes into effect

File: addons/sourcemod/configs/vsh/vsh.cfg [hunk 2]
```diff
@@ -1334,7 +1334,9 @@
 		
 		"327"	//Claidheamh Mòr
 		{
-			"desp"			"Claidheamh Mòr: {positive}Refills 25% of your charge meter on hit"
+			"desp"			"Claidheamh Mòr: {positive}Refills 25% of your charge meter on hit, {neutral}replaces damage vulnerability with slower attack speed"
+			"attrib"		"5 ; 1.15"	
+
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/configs/vsh/vsh.cfg:1340]: Extra space

File: addons/sourcemod/configs/vsh/vsh.cfg (hunk 1 above)
Comment (StarInAPond) [reply addons/sourcemod/configs/vsh/vsh.cfg]: added

**Review by FortyTwoFortyTwo — APPROVED** [review]

**Pull request merged**